src/main/java/com/dumbdogdiner/stickyapi/bukkit/util/ServerUtil.java

```diff
@@ -4,19 +4,17 @@
  */
 package com.dumbdogdiner.stickyapi.bukkit.util;
 
-import javax.annotation.Nullable;
-
 import com.destroystokyo.paper.Title;
-
 import com.dumbdogdiner.stickyapi.common.translation.Translation;
 import com.dumbdogdiner.stickyapi.common.util.reflection.ReflectionUtil;
 
+import net.md_5.bungee.api.chat.TextComponent;
 import org.bukkit.Bukkit;
 import org.bukkit.Server;
 import org.bukkit.entity.Player;
 import org.jetbrains.annotations.NotNull;
 
-import net.md_5.bungee.api.chat.TextComponent;
+import javax.annotation.Nullable;
```

Comment:
Wrong annotation! We use IntelliJ's ones, although is worth considering using Lombok's since they do add some extra runtime safety.
```suggestion
import whatever.the.intellij.package.is.Nullable;
```